lumigator/python/mzai/backend/backend/tests/api/routes/test_health.py

```diff
@@ -4,6 +4,29 @@
 from fastapi import status
 from fastapi.testclient import TestClient
 from schemas.extras import HealthResponse
+from schemas.jobs import JobCreate, JobResponse
+from schemas.datasets import DatasetFormat, DatasetResponse
+
+import pytest
+import csv
+import io
+
+
+def format_dataset(data: list[list[str]]) -> str:
+    """Format a list of tabular data as a CSV string."""
+    buffer = io.StringIO()
+    csv.writer(buffer).writerows(data)
+    buffer.seek(0)
+    return buffer.read()
+
+
+@pytest.fixture
+def valid_experiment_dataset() -> str:
+    data = [
+        ["examples", "ground_truth"],
+        ["Hello World", "Hello"],
+    ]
+    return format_dataset(data)
```

Comment:
We have identical code in [test_datasets](https://github.com/mozilla-ai/lumigator/blob/main/lumigator/python/mzai/backend/backend/tests/api/routes/test_datasets.py), what about putting the common stuff in [conftest](https://github.com/mozilla-ai/lumigator/blob/main/lumigator/python/mzai/backend/backend/tests/conftest.py) instead?

Reply:
Should we make a standalone conftest for api tests alone?

Reply:
Ok, let's keep a single one for the moment.